zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLFuncVisitor.scala

```diff
@@ -190,16 +202,25 @@ object AQLFuncVisitor {
   def translateAggrFunc(field: Field,
                         func: AggregateFunc,
                         aqlExpr: String
-                       ): (DataType.DataType, String) = {
+                       ): (DataType.DataType, String, String, String) = {
```

Comment:
since, it returns multiple same type object (`String`s), it's better to write a java-doc to explain the returns.